.env.benchmark.example

```
@@ -0,0 +1,11 @@
+# Путь к конфигурации EDT (можно заменить на свою)
+BENCH_EDT_PATH=src/test/resources/ext/edt/ssl_3_1/configuration
+
+# Путь к конфигурации Designer
+BENCH_DESIGNER_PATH=src/test/resources/ext/designer/ssl_3_1/src/cf
+
+# Дополнительные JVM-аргументы для JMH
+BENCH_JVM_ARGS=-Xms4g -Xmx8g
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)